cmd/kube-apiserver/app/options/features.go

```diff
@@ -0,0 +1,18 @@
+package options
+
+import (
+	genericfeatures "k8s.io/apiserver/pkg/features"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/features"
+)
+
+var featureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
+	features.ExternalTrafficLocalOnly: {Default: true, PreRelease: utilfeature.GA},
```

Comment:
> If a component registers a previously set feature flag with the same values, the current behaviour is to skip it. If there's any deviations, e.g. in terms of conflicting default values, an error is raised.

But doesn't this PR make it so separate components don't register into the same set, and therefore wouldn't notice conflicting defaults?